frontend/src/components/UserCard.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component, PropTypes} from 'react';
+import UserPhoto from './UserPhoto';
+import moment from 'moment';
+
+export default class UserCard extends Component {
+  static propTypes = { user: PropTypes.object };
+
+  static defaultProps = {
+    user: {
+      avatar: '/static/images/default_avatar.svg',
+      name: '',
+      role: '',
+      tier: '',
+      totalDonations: 0,
+      twitterHandle: '',
+      website: ''
+    }
+  };
+
+  _link(href, children) {
```

Comment:
This function (`_link`) is currently being used on the `UserList` component too, but that component will eventually being deprecated once the new design is accepted.